packages/block-editor/src/components/block-list/style.scss

```diff
@@ -614,28 +614,58 @@
  */
 
 .block-editor-block-list__block-selection-button {
-	display: block;
+	display: inline-flex;
+	padding: 0 20px;
```

Comment:
This was kinda too hard to get right, none of the `$grid-unit` options seemed to fit right "visually" with the chip once the drag started.